06_puppy/hsy3418/syncStore.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"fmt"
+	"sync"
+)
+
+type SyncStore struct {
```

Comment:
You have done really well documenting your interface methods, but you forgot these as well. Since the SyncStore and NewSyncStore are also included as part of the exported package, these should also have documentation around them.